packages/linux-drivers/ssv6xxx/package.mk

```diff
@@ -0,0 +1,65 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="ssv6xxx"
+PKG_VERSION="1041e7d"
+PKG_SHA256="f983aef19014f23536a6327d026f92a39d3af14f7001ee9d70ab696f8382b20a"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="http://libreelec.tv"
+PKG_URL="http://kszaq.libreelec.tv/sources/ssv6xxx-$PKG_VERSION.tar.gz"
```

Comment:
`$DISTRO_SRC/$PKG_NAME-$PKG_VERSION.tar.xz`
 drop `PKG_SOURCE_DIR="ssv6xxx-${PKG_VERSION}*"`
 change `PKG_SHA256="d7213b7a27cbca686b7be787cf54220adc180716ad03cbc1eca6e51e592344ac"`

Reply:
this change causes the following problem

```[adamg]@[ubuntu] ~/repos/LibreELEC $ scripts/build ssv6xxx
  UNPACK   ssv6xxx
scripts/unpack: line 91: [: /home/adamg/repos/LibreELEC/build.LibreELEC-S905.aarch64-9.0-devel/ssv6xxx-1041e7d: binary operator expected
      APPLY PATCH (common)   packages/linux-drivers/ssv6xxx/patches/ssv6xxx-aml-001-fix-build-and-firmware-path.patch
patch: **** Can't change to directory /home/adamg/repos/LibreELEC/build.LibreELEC-S905.aarch64-9.0-devel/ssv6xxx-1041e7d : Not a directory
[adamg]@[ubuntu] ~/repos/LibreELEC $```